plugins/tiddlywiki/markdown/EditorToolbar/mono-block.tid

```diff
@@ -0,0 +1,17 @@
+title: $:/plugins/tiddlywiki/markdown/EditorToolbar/mono-block
+list-after: $:/core/ui/EditorToolbar/mono-block
+tags: $:/tags/EditorToolbar
+icon: $:/core/images/mono-block
+caption: {{$:/language/Buttons/MonoBlock/Caption}} (Markdown)
+description: {{$:/language/Buttons/MonoBlock/Hint}}
+condition: [<targetTiddler>type[text/x-markdown]]
```

Comment:
Condition should be like this: `[<targetTiddler>type[text/x-markdown]] [<targetTiddler>type[text/markdown]]` since both types are possible now. Also see: https://github.com/Jermolene/TiddlyWiki5/pull/6528

Reply:
OK, I have changed it.